public/API/API_GetGame.php

```diff
@@ -48,5 +49,6 @@
     'Publisher' => $game->Publisher,
     'Developer' => $game->Developer,
     'Genre' => $game->Genre,
-    'Released' => $game->Released,
+    'Released' => $game->released_at?->format('Y-m-d H:i:s'),
```

Comment:
Why include hours/minutes/seconds if the granularity doesn't go that far? It looks like the time portion comes from when the released_at was last updated:
```
"Released":"1985-09-13 21:29:49","ReleasedAtGranularity":"day"
```

```
+---------------------+---------------------+
| released_at         | updated             |
+---------------------+---------------------+
| 1985-09-13 21:29:49 | 2024-08-15 21:29:49 |
+---------------------+---------------------+
```

If it's desirable to keep a full timestamp, suggesting forcing to midnight.

Reply:
Good point -- set to YYYY-MM-DD in latest.